deno/payloads/v10/user.ts

```diff
@@ -87,8 +87,16 @@ export interface APIUser {
 	 * The user's avatar decoration hash
 	 *
 	 * See https://discord.com/developers/docs/reference#image-formatting
+	 *
+	 * @deprecated Use `avatar_decoration_data` instead
 	 */
 	avatar_decoration?: string | null;
```

Comment:
the old avatar_decoration field does not exist anymore (and has not existed for a long time) so we could just delete it, or leave it deprecated

Reply:
Deprecated and always null